Source/player.cpp

```diff
@@ -444,12 +460,13 @@ static DWORD GetPlrGFXSize(HeroClass c, const char *szCel)
 	for (a = &ArmourChar[0]; *a; a++) {
 		if (gbIsSpawn && a != &ArmourChar[0])
 			break;
-		for (w = &WepChar[0]; *w; w++) { // BUGFIX loads non-existing animagions; DT is only for N, BT is only for U, D & H (fixed)
+		for (w = &WepChar[0]; *w; w++) { // BUGFIX loads non-existing animagions; DT is only for N, BL is only for U, D & H (fixed)
 			if (szCel[0] == 'D' && szCel[1] == 'T' && *w != 'N') {
 				continue; //Death has no weapon
 			}
 			if (szCel[0] == 'B' && szCel[1] == 'L' && (*w != 'U' && *w != 'D' && *w != 'H')) {
-				continue; //No block without weapon
+				if (c != HeroClass::Monk || (*w != 'T' && *w != 'N')) // monk can block with a staff and unarmed

```

Comment:
What if the monk has a single one-handed weapon equiped?

Reply:
then he can't block?

Reply:
> A Monk can also block with a staff and with at least one hand bare.

http://www.bigd-online.com/JG/JGFrame.html

Section 2.2.1

Reply:
jesus, so when he CAN'T block? lol